packages/webapp/src/webview/ui/components/GuidedAnswerNavPath/GuidedAnswerNavPath.tsx

```diff
@@ -15,8 +16,17 @@ import { focusOnElement } from '../utils';
 export function GuidedAnswerNavPath(): ReactElement {
     let firstTimeFocus: boolean;
     const nodes = useSelector<AppState, GuidedAnswerNode[]>((state) => state.activeGuidedAnswerNode);
-    const activeNode = nodes[nodes.length - 1];
-    const lastIndex = nodes.length - 1;
+    const isSolved = useSelector<AppState, boolean | null>((state) => state.guideFeedback);
+    let activeNode = nodes[nodes.length - 1];
+    let lastIndex = nodes.length - 1;
+    let lastBlockBorderStyle = 'timeline-content-bottom-border';
+
+    if (isSolved === false && isSolved !== null) {
+        nodes.push({ NODE_ID: 99999, TITLE: i18next.t('ISSUE_IS_NOT_RESOLVED'), BODY: '', QUESTION: '', EDGES: [] });
```

Comment:
I would not add something to the state here, `GuidedAnswerNavPath` should not take care of adding this unresolved node. 

Instead I'd add it to reducer here:
[GuidedAnswerNavPath
](https://github.com/SAP/guided-answers-extension/pull/259/files#diff-4a3433922f8d94c2db6d305df038553fcd22b858d29fb53394b117f5a003be1fR116-R119)

Instead of 
```typescript
case GUIDE_FEEDBACK: {
    newState.guideFeedback = action.payload;
    break;
}
```

I'd do 

```typescript
case GUIDE_FEEDBACK: {
    newState.guideFeedback = action.payload;
    if (action.payload === false) {
        newState.activeGuidedAnswerNode.push({
            NODE_ID: 99999,
            TITLE: i18next.t('ISSUE_IS_NOT_RESOLVED'),
            BODY: '',
            QUESTION: '',
            EDGES: []
        });
    }
    break;
}
```

Reply:
Thanks Klaus, that makes sense. I'll add that change